CONTRIBUTING.md

```diff
@@ -12,13 +12,19 @@ Once a commit is made to `develop`, a PR should be opened from `develop` into `m
 
 # Unit Testing
 
-TL;DR
+When contributing to WPCS, it's recommended that the project has its own copy of `phpcs`, installed via Composer:
```

Comment:
This is only valid if your preferred method of installation is Composer. The PEAR install comes with the test suite already and if you also develop for PHPCS itself, you probably have a stand-alone PHPCS git clone somewhere which comes with the test suite as well.